recipe/meta.yaml

```diff
@@ -1,51 +1,53 @@
-{% set version = "1.0.5" %}
-{% set sha256 = "c8459c15b75a65f2f468fd0a372cf3bc2262262fd6277054de7c2029bff64678" %}
+{% set name = "prompt_toolkit" %}
+{% set version = "1.0.6" %}
+{% set sha256 = "fa8f0c3668afbc85abbb4601ebc8db1db9d5009c8a833feea107fc63f1e7ef22" %}
 
 package:
-    name: prompt_toolkit
-    version: {{ version }}
+  name: {{ name }}
+  version: {{ version }}
 
 source:
-    fn: python-prompt-toolkit-{{ version }}.tar.gz
-    url: https://github.com/jonathanslenders/python-prompt-toolkit/archive/{{ version }}.tar.gz
-    sha256: {{ sha256 }}
+  fn: python-prompt-toolkit-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
They didn't have a tag at the time I wrote this. So I asked them for one in issue ( https://github.com/jonathanslenders/python-prompt-toolkit/issues/381 ). Now that there is one, I'm more than happy to revert back to the GitHub tag.